pbcc.py

```diff
@@ -5,8 +5,9 @@
 
 
 def get_list(item, index):
```

Comment:
this is same function. Better to move in new file like utils.py to avoid code duplication. Then import utils whenever this function is needed